### PR TITLE
[Form] Added missing Spanish translations

### PR DESCRIPTION
**Pull request by TavoNiievez**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Part of #38710 and #38764
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]
Nice work

Thank you

File: src/Symfony/Component/Form/Resources/translations/validators.es.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter an integer.</source>
+                <target>Por favor, ingrese un número entero.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Por favor, seleccione un idioma válido.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Por favor, seleccione un lugar válido.</target>
```
Comment (TavoNiievez) [inline src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: @Nyholm while checking this: https://github.com/symfony/symfony/pull/38721#discussion_r511602598,
I realized that this translation for 'locale' is for 'location' as well.
In Spanish, and in this context, 'locale' refers to 'configuración regional de idioma' = 'language locale'.
is it ok if i interpret this as 'Please select a valid language'?

Comment (derrabus) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: No, that would be wrong. A locale is rather some region-specific setting. It includes language but also formatting of numbers, dates, time, currency etc. as well as string collations and character case conversions.

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: So a locale is "sv", "en", "es" etc. 
A language is "Swedish", "English", "Spanish". 

The word "locale" is a technical word, that is fine.

In the Arabic language, there is apparently no word for "locale", so we fallback to "language". I would much prefer if we used the Spanish word for "locale" here. 

Using "location", (Stockholm, "at work" or "London") is the wrong word in this context.

Comment (TavoNiievez) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: @derrabus OK! thanks! if that's the case the correct translation would be 'Por favor, seleccione una configuración regional válida'.

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: Yes. I agree. Can you please update?

Comment (TavoNiievez) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf:75]: OK. Just to be clear about the changes:
If we talk about 'configuración regional' i would expect to see something like 'es_ES' = 'Spanish Spain' or 'es_CO' = 'Spanish Colombia', and if we talk about 'language' it is only 'es' = 'Spanish'.
i will update it.

Comment (TavoNiievez) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: OK. Just to be clear about the changes:
If we talk about 'configuración regional' i would expect to see something like 'es_ES' = 'Spanish Spain' or 'es_CO' = 'Spanish Colombia', and if we talk about 'idioma' it is only 'es' = 'Spanish'.

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: That is way too much Spanish for me (even with Google translate) I'll let someone else give their opinion. 

But both are better alternatives that the current one.

Comment (TavoNiievez) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: ok, i'll wait for more feedback
**Edit:** [Some Spanish websites](https://diego.com.es/establecer-el-locale-del-usuario-en-symfony) do not translate this term. They simply refer to it as 'el *locale*'.

Comment (javiereguiluz) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: @TavoNiievez please, use this translation recommended by Wikipedia -> https://es.wikipedia.org/wiki/Configuraci%C3%B3n_regional

**Review by javiereguiluz — APPROVED** [review]
Good work! Thanks @TavoNiievez

File: src/Symfony/Component/Form/Resources/translations/validators.es.xlf (hunk 1 above)
Comment (TavoNiievez) [reply src/Symfony/Component/Form/Resources/translations/validators.es.xlf]: @javiereguiluz thanks you! Done. 😃

**Comment by Nyholm** [comment]
Thanks @TavoNiievez.

**Pull request merged**